tests/e2e/specs/student-test.js

```diff
@@ -11,14 +17,14 @@ describe('Mesh landing page after login', () => {
 
   it('Shows module stuff after clicking on a button', () => {
     cy.get('[data-cy=mod1]').click()
-    cy.url().should('eq', 'http://localhost:8081/modules/1')
+    cy.url().should('eq', 'http://localhost:8080/modules/1')
   })
 })
 
 // this one isn't working for some reason. I think I need to change the route or test the route by clicking???
 describe('Mesh about page', () => {
-  it.skip('Visits the about route and renders without errors', () => {
-    cy.visit('/about')
-    cy.get('.feedback-container').children('.about').should('exist')
+  it('Visits the about route and renders without errors', () => {
+    cy.visit('http://localhost:8080/about')
+    cy.get('h1').should('exist')
```

Comment:
We can start writing real tests as soon as our UI has all the kinks worked out. Hopefully today-tomorrow-ish. Thank you so much for your hard work on TravisCI!